packages/desktop/components/popups/AllotManaPopup.svelte

```diff
@@ -130,7 +141,9 @@
                     submitHandler={onSubmit}
                     disabled={isBusy}
                 />
-                <ManaBox {transactionInfo} bind:hasEnoughMana />
+                {#if displayManaBox}
+                    <ManaBox {transactionInfo} bind:hasEnoughMana />
+                {/if}
             </div>
             <popup-buttons class="flex flex-row flex-nowrap w-full space-x-4">
                 <Button classes="w-full" outline onClick={onBackClick} disabled={isBusy}
```

Comment:
if there is no amount and no account address, I would disable this button :pray: